pkg/registry/core/podtemplate/strategy.go

```diff
@@ -67,7 +69,8 @@ func (podTemplateStrategy) AllowCreateOnUpdate() bool {
 
 // PrepareForUpdate clears fields that are not allowed to be set by end users on update.
 func (podTemplateStrategy) PrepareForUpdate(ctx genericapirequest.Context, obj, old runtime.Object) {
-	_ = obj.(*api.PodTemplate)
+	template := obj.(*api.PodTemplate)
+	pod.DropDisabledAlphaFields(&template.Template.Spec)
 }
```

Comment:
I think registry/core/*pod*/strategy.go is the one you want

Reply:
And you need to drop from the old object as well

Reply:
Nice!

Reply:
Do I need to remove these changes from podtemplate package? I think it will work for both.

Reply:
podtemplates are not actually enabled yet. In the interest of a minimal change for a maintenance branch, I would remove it

Reply:
Changed as you request